app/Database/Requirement.hs

```diff
@@ -14,35 +14,35 @@ module Database.Requirement
 import qualified Data.String as S
 import qualified Data.List as L
 
-data ProgramReq = PRGREQ String [Req]
+data ProgramReq = PRGREQ String [Req] deriving (Eq, Show)
 
 -- Returns a well formatted String representing a program requirement for specified program.
-instance Show ProgramReq where
-    show (PRGREQ program reqs) = "Program Requirements for " ++ program
-                               ++ ":\n" ++ L.intercalate "\n" (map show reqs)
+-- instance Show ProgramReq where
```

Comment:
Eh, I think it's actually better to remove `ProgramReq` and `CourseReq` altogether. They won't be necessary for the final use of this parser.